pkg/api/validate.go

```diff
@@ -164,7 +164,7 @@ func validateProperties(p *Properties, location string, externalOnly bool) (errs
 
 	errs = append(errs, validateProvisioningState(p.ProvisioningState)...)
 	switch p.OpenShiftVersion {
-	case "v3.10":
+	case "v3.11":
```

Comment:
We won't be able to test published 3.11 images once this gets merged so it probably makes sense to continue supporting 3.10 until we have a published 3.11 image?

Reply:
I think we might want to do a switch. I would like to merge this only when we have images published and we can start iterate on this. We can discuss.